aisim/atoms.py

```diff
@@ -141,7 +155,7 @@ def calc_position(self, t):
         return self.initial_position + self.initial_velocity * t
 
     def state_occupation(self, state):
-        return np.abs(self.state_vectors[:, state])**2
+        return np.real(np.einsum('ijk,ijk->ij' , self.state_bras, self.state_kets)[:,state])

```

Comment:
trailing whitespace after string

will try to setup automatic checking with flake8 in PRs soon...